app/src/main/java/com/andrew/tasky/domain/StringToInitials.kt

```diff
@@ -0,0 +1,24 @@
+package com.andrew.tasky.domain
+
+class StringToInitials {
```

Comment:
Could be an object as well, since you don't need to pass any parameters to this class